ajax/libs/Bounce/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bounce",
+  "filename": "bounce.min.js"
+  "version": "0.0.0",
+  "description": "Bounce.js is a tool and JS library for generating beautiful CSS3 powered keyframe animations.",
+  "homepage": "http://bouncejs.com",
+  "keywords": [
+    "bounce",
+    "bounce.js",
+    "bouncejs.com"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tictail/bounce.js/"
```

Comment:
Let's use `https://github.com/tictail/bounce.js` as the url value.
Because this URL can be accessed by people, the `/` of end line is not necessary.
Thank you.